### PR TITLE
Feature/acm integration

### PR DESCRIPTION
**Pull request by shapirov103**
*Issue #, if available:*

*Description of changes:*
ACM integration
Prepared to migrate to aws-quickstart npm repo
Fixed a number of issues integrating with latest changes from the SSP quickstart.
Nginx blueprint with ACM integration works e2e (workload PR is coming)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/bottlerocket-construct/index.ts [hunk 1]
```diff
@@ -1,11 +1,14 @@
 import * as cdk from '@aws-cdk/core';
 
 // SSP Lib
-import * as ssp from '@shapirov/cdk-eks-blueprint'
+import * as ssp from '@aws-quickstart/ssp-amazon-eks'
```
Comment (askulkarni2) [inline lib/bottlerocket-construct/index.ts:4]: Missing `;`

File: lib/custom-cluster-construct/index.ts [hunk 2]
```diff
@@ -2,10 +2,11 @@ import * as cdk from '@aws-cdk/core';
 import * as ec2 from '@aws-cdk/aws-ec2';
 import * as eks from '@aws-cdk/aws-eks';
 // SSP Lib
-import * as ssp from '@shapirov/cdk-eks-blueprint'
+import * as ssp from '@aws-quickstart/ssp-amazon-eks'
```
Comment (askulkarni2) [inline lib/custom-cluster-construct/index.ts:5]: Missing `;`

File: lib/fargate-construct/index.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import * as cdk from '@aws-cdk/core';
 import * as eks from '@aws-cdk/aws-eks';
 // SSP Lib
-import * as ssp from '@shapirov/cdk-eks-blueprint'
+import * as ssp from '@aws-quickstart/ssp-amazon-eks'
```
Comment (askulkarni2) [inline lib/fargate-construct/index.ts:4]: Missing `;`

File: lib/multi-team-construct/index.ts [hunk 4]
```diff
@@ -1,7 +1,8 @@
 import * as cdk from '@aws-cdk/core';
 
 // SSP Lib
-import * as ssp from '@shapirov/cdk-eks-blueprint'
+import * as ssp from '@aws-quickstart/ssp-amazon-eks'
```
Comment (askulkarni2) [inline lib/multi-team-construct/index.ts:4]: Missing `;`

File: lib/nginx-ingress-construct/index.ts [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+// SSP Lib
+import * as ssp from '@aws-quickstart/ssp-amazon-eks'
+
+//TODO import * as iam from '@aws-cdk/aws-iam';
+// import * as route53 from '@aws-cdk/aws-route53';
+
+
+// Team implementations
+import * as team from '../teams'
```
Comment (askulkarni2) [inline lib/nginx-ingress-construct/index.ts:11]: Missing `;`

File: lib/pipeline-stack/index.ts [hunk 6]
```diff
@@ -2,7 +2,8 @@ import * as cdk from '@aws-cdk/core';
 import { StackProps } from '@aws-cdk/core';
 
 // SSP Lib
-import * as ssp from '@shapirov/cdk-eks-blueprint'
+import * as ssp from '@aws-quickstart/ssp-amazon-eks'
```
Comment (askulkarni2) [inline lib/pipeline-stack/index.ts:5]: Missing `;`

**Review by askulkarni2 — DISMISSED** [review]
LGTM! Just a few missing `;`

**Comment by askulkarni2** [comment]
@shapirov103 there is a CI check failing as well.

**Comment by shapirov103** [comment]
> @shapirov103 there is a CI check failing as well.

Yeah, it is because I used the tar for the npm module since we have not published the quickstart yet.

**Review by askulkarni2 — APPROVED** [review]
LGTM!

File: lib/bottlerocket-construct/index.ts (hunk 1 above)
Comment (shapirov103) [reply lib/bottlerocket-construct/index.ts:4]: as we discussed, let's table it for now, there are a bunch of other places outside of this PR that will need clean up.  we will get to it at some point.

**Pull request merged**